tests/meson.build

```diff
@@ -40,19 +46,29 @@ foreach pair : test_programs
 endforeach
 
 foreach test_script : test_scripts
+  if test_script.endswith('.py')
+    interpreter = python
+  else
+    interpreter = bash
+  endif
+
+  if not interpreter.found()
+    continue
+  endif
```

Comment:
Is it a good idea to skip tests because the test interpreter isn't found?

Can it ever not be found? Meson's reference implementation is python, so that will always be found (and meson's own sys.executable is the fallback if python3 is not in $PATH) and bash is typically very unlikely to not be installed on Linux.

Also -- you can just directly run the test scripts as your test() executable, they are all executable in git and can be run as programs without a script wrapper.

Reply:
> Is it a good idea to skip tests because the test interpreter isn't found?

In general, I'd rather have less test coverage (in a backports to ancient distributions, for instance) than preventing backporters from running tests at all. The test for multiple seccomp programs automatically skips itself if python3-seccomp isn't found, because some older distributions don't have that module available.

Having said that, yes, for these specific interpreters, there's probably little point in skipping the tests.

Reply:
The reason I'm explicitly invoking an interpreter is that one of the environments I'm compiling bubblewrap for is the Steam Runtime, based on a very old version of Ubuntu, which has a symlink `python3 -> python3.2`, a separate `python3.5` executable, and a version of Meson that specifically runs under `python3.5`. Explicitly specifying the interpreter gives it the opportunity to have a build option for which Python to use.

Reply:
Sure, I get the idea of marking tests as skipped. Indeed, the seccomp test handles that via a `# SKIP` directive and the test harness even records that as a skipped test. :D

It's less clear what to do with missing interpreters (if it were indeed possible to not have them) and I don't really have a good answer for that when you can't even run the test to emit `# SKIP`.